src/main/resources/log4j2.xml

```diff
@@ -0,0 +1,14 @@
+<Configuration packages="ch.hevs.cloudio.endpoint">
```

Comment:
How will this work if the application using the endpoint library has it's own log configuration? Is there no way to add the CloudioLogAppender at runtime?

Reply:
Will try to do the full configuration by code

Reply:
Did add the CloudioLogAppender config in code, tested with a custom configuration in a project and both work together like a charm.